app/src/main/java/com/iron/espresso/presentation/home/study/adapter/viewholder/PlaceViewHolder.kt

```diff
@@ -0,0 +1,27 @@
+package com.iron.espresso.presentation.home.study.adapter.viewholder
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.databinding.DataBindingUtil
+import androidx.recyclerview.widget.RecyclerView
+import com.iron.espresso.R
+import com.iron.espresso.model.response.Place
+
+class PlaceViewHolder(parent: ViewGroup) : RecyclerView.ViewHolder(
+    LayoutInflater.from(parent.context).inflate(R.layout.item_place_list, parent, false)
+) {
+    private val binding =
+        DataBindingUtil.bind<com.iron.espresso.databinding.ItemPlaceListBinding>(itemView)
+
+    fun bind(item: Place, itemClickListener: (item: Place) -> Unit) {
```

Comment:
```suggestion
class PlaceViewHolder(parent: ViewGroup, private val itemClickListener: (item: Place) -> Unit) : RecyclerView.ViewHolder(
    LayoutInflater.from(parent.context).inflate(R.layout.item_place_list, parent, false)
) {
    private val binding =
        DataBindingUtil.bind<com.iron.espresso.databinding.ItemPlaceListBinding>(itemView)

    fun bind(item: Place) {
```
요러케 정리하는게 의미에 맞을 것 같아요

Reply:
요기도 수정했습니다.

Reply:
![image](https://user-images.githubusercontent.com/54328309/99685147-9adc8800-2ac5-11eb-81d9-34b33afe93eb.png)
@4147b47